src/components/Channel/Channel.js

```diff
@@ -566,11 +473,98 @@ const ChannelInner = ({
     );
   }
 
-  return (
-    <div className={`str-chat str-chat-channel ${chatContext.theme}`}>
-      {core}
-    </div>
-  );
+  return <div className={`str-chat str-chat-channel ${theme}`}>{core}</div>;
+};
+
+Channel.defaultProps = {
+  multipleUploads: true,
+};
+
+Channel.propTypes = {
+  /** Which channel to connect to, will initialize the channel if it's not initialized yet */
+  channel: PropTypes.instanceOf(StreamChannel),
+  /**
+   * Empty channel UI component. This will be shown on the screen if there is no active channel.
+   *
+   * Defaults to null which skips rendering the Channel
+   *
+   * */
+  EmptyPlaceholder: PropTypes.element,
+  /**
+   * Error indicator UI component. This will be shown on the screen if channel query fails.
+   *
+   * Defaults to and accepts same props as: [LoadingErrorIndicator](https://getstream.github.io/stream-chat-react/#loadingerrorindicator)
+   *
+   * */
+  // @ts-ignore elementType
```

Comment:
changed these within my immutable removal. `@ts-ignore` should not be used basically ever, it should be `@ts-expect-error` until we fix the error as tsc will then let us know this annotation is still on when it is no longer needed